package.json

```diff
@@ -11,7 +11,8 @@
   },
   "scripts": {
     "update-retiring": "node bin/update-retiring",
-    "pull": "node bin/update-firestore-from-propublica",
+    "seed-federal": "node bin/seed-federal-db-from-propublica.js",
+    "pull": "node bin/update-firestore-from-propublica.js",
```

Comment:
Because there are two data streams now, can this be renamed to `update-from-propublica`?

Reply:
yeah